tests/models/speecht5/test_tokenization_speecht5.py

```diff
@@ -52,12 +52,24 @@ def get_input_output_texts(self, tokenizer):
         output_text = "this is a test"
         return input_text, output_text
 
+    def get_numeric_input_output_texts(self):
```

Comment:
Nice - thanks for checking all cases!

Reply:
I am guessing that `"8,000"` is for American normalizer? Or should it also be included?

Reply:
Sorry, I didn't quite understand what you were referring to @ArthurZucker

Reply:
Just saying that some might write `8000` as `8,000` and wondering if this should also be accounted for?

Reply:
Got it! Adding the below line in the next commit which deals with this issue and normalizes any number with comma separation.

```python
text = re.sub(r"(\d+,\d+)", lambda match: match.group(1).replace(",", ""), text)
```